src/websocket.ts

```diff
@@ -2,7 +2,9 @@ const host =
   process.env.NODE_ENV === "production"
     ? window.location.host
     : "localhost:3005";
-const { io } = require("socket.io-client");
+
+import {io} from 'socket.io-client';
+// const { io } = require("socket.io-client");
```

Comment:
Remove this.

Reply:
You are asking me to remove the comment right?